migration-guide.adoc

```diff
@@ -48,3 +48,10 @@ MyEntity myEntity = session.find(MyEntity.class, 2, LockMode.WRITE);
 ```
 
 only the entity with id equals to 2 will be loaded but the proxy will not be initialized.
+
+[[maven-metamodel-generation]]
+== Integrating Static Metamodel Generation in Maven Projects
+The integration of the static metamodel generation in a maven project has changed; the recommended way to do this now is through the annotation processor path of the maven compiler plugin.
+Check out the {userGuideBase}#tooling-maven-modelgen[User Guide] for an example of this.
```

Comment:
It can work differently in Gradle as well if they happened to be putting `hibernate-jpamodelgen` on the compile classpath rather than the processorpath.  I'd prefer if this mentions both and point to the chapter or to each section.